src/js/components/atoms/input-checkbox/input-checkbox.ts

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { InputCheckboxProps } from './input-checkbox.models';
+import WithRender from './input-checkbox.vue.html';
+
+@WithRender
+@Component({
+    name: 'InputCheckbox',
+})
+export class InputCheckbox extends Vue implements InputCheckboxProps {
+    @Prop({ type: Boolean, default: false }) checked?: InputCheckboxProps['checked'];
+    @Prop({ type: Boolean, default: false }) disabled?: InputCheckboxProps['disabled'];
+    @Prop({ type: String, required: true }) label!: InputCheckboxProps['label'];
+    @Prop({ type: String }) name!: InputCheckboxProps['name'];
+    @Prop({ type: String }) value?: InputCheckboxProps['value'];
```

Comment:
Is the checkbox used for more components? Otherwise, I would suggest not making the value optional given it makes a checkbox a bit useless.

Reply:
done